application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -134,34 +136,40 @@ private boolean isUserOnCooldown(User user) {
 
     private void sendCooldownResponse(SlashCommandInteractionEvent event) {
         User user = event.getUser();
+        Guild guild = event.getGuild();
 
-        String message =
-                """
-                        Sorry, you can only create a single help thread every 5 minutes. Please use your existing thread %s instead.
-                        If you made a typo or similar, you can adjust the title using the command %s and the category with %s :ok_hand:""";
-
-        RestAction<String> changeTitle = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
-                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
-                HelpThreadCommand.CHANGE_TITLE_SUBCOMMAND);
-        RestAction<String> changeCategory = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
-                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
-                HelpThreadCommand.CHANGE_CATEGORY_OPTION);
-        long lastCreatedThreadId = database
-            .read(context -> context.selectFrom(HELP_THREADS)
-                .where(HELP_THREADS.AUTHOR_ID.eq(user.getIdLong()))
-                .orderBy(HELP_THREADS.CREATED_AT.desc())
-                .fetch())
-            .get(0)
-            .getChannelId();
-
-        if (lastCreatedThreadId == 0) {
+        RestAction<String> changeTitle = mentionHelpChangeCommand(guild, CHANGE_TITLE_SUBCOMMAND);
+        RestAction<String> changeCategory =
+                mentionHelpChangeCommand(guild, CHANGE_CATEGORY_SUBCOMMAND);
+
+        HelpThreadsRecord lastThreadByAuthor =
+                database.read(context -> context.selectFrom(HELP_THREADS)
+                    .where(HELP_THREADS.AUTHOR_ID.eq(user.getIdLong()))
+                    .orderBy(HELP_THREADS.CREATED_AT.desc())
+                    .limit(1)
+                    .fetchOne());
```

Comment:
can u move this into a helper method please?

Reply:
done